### PR TITLE
Moved schema version definition into db package

### PR DESCRIPTION
**Pull request by gentlecat**

### CODE REVIEW

File: db/__init__.py [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 import logging
 
+
+# This value must be incremented after schema changes on replicated tables!
+__version__ = 2
```
Comment (alastair) [inline db/__init__.py]: Should we call this `version`, or `schema`?

Comment (gentlecat) [inline db/__init__.py]: `SCHEMA_VERSION`?

Comment (alastair) [inline db/__init__.py]: ok!

Comment (gentlecat) [inline db/__init__.py]: Done.

**Pull request merged**